locked-asset/factory/src/locked_asset.rs

```diff
@@ -132,7 +132,7 @@ pub trait LockedAssetModule: token_supply::TokenSupplyModule + token_send::Token
         let mut percents_sum: u8 = 0;
         let mut last_milestone_unlock_epoch: u64 = 0;
 
-        for milestone in unlock_milestones.into_iter() {
+        for milestone in unlock_milestones.to_vec().into_iter() {
```

Comment:
is the above `unlock_milestones.into_iter()` the same as this one? (`unlock_milestones.to_vec().into_iter()`)

Reply:
Yes